### PR TITLE
Update mkdocs.yml

### PR DESCRIPTION
**Pull request by ContentBrewmaster**
Fixing navigation to make Canadian page label equivalent to US version.

----
:books: Documentation preview :books:: https://civicactions-handbook--1178.org.readthedocs.build/en/1178/

### CODE REVIEW

File: employee-benefits/pto-canada.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Benefits policy (Canadian employees)
```
Comment (dmundra) [inline employee-benefits/pto-canada.md]: @ContentBrewmaster can you update this title to match the US version as well?

**Review by dmundra — CHANGES_REQUESTED** [review]
Looks good to me. Noticed one small change.

File: employee-benefits/pto-canada.md (hunk 1 above)
Comment (ContentBrewmaster) [reply employee-benefits/pto-canada.md]: @dmundra Done!

**Review by dmundra — APPROVED** [review]
Changes look good to me. Thank you @ContentBrewmaster. Over to @grugnog for additional review.

**Comment by ContentBrewmaster** [comment]
Not sure how to fix redirects file @dmundra

**Comment by dmundra** [comment]
@ContentBrewmaster fixed it. This one or #1178 will have the same conflict occur when one them is merged and I can help resolve that as well.

File: .config/mkdocs.yml [hunk 2]
```diff
@@ -19,10 +19,10 @@ nav:
       - about-civicactions/annual-summit.md
   - Employee benefits:
       - employee-benefits/README.md
-      - Personal Time Off (PTO): employee-benefits/README.md
+      - Personal Time Off (US employees): employee-benefits/README.md
       - employee-benefits/us-compensation.md
       - employee-benefits/us-tech-stipend.md
-      - employee-benefits/canada-benefits-policy.md
+      - Personal Time Off (Canadian employees): employee-benefits/pto-canada.md
```
Comment (grugnog) [inline .config/mkdocs.yml]: @ContentBrewmaster I am not clear why we need the titles here, if they are the same as the page headers?

Comment (ContentBrewmaster) [reply .config/mkdocs.yml]: @grugnog you reminded me that the new title on each page is populating the nav.

Comment (ContentBrewmaster) [reply .config/mkdocs.yml]: @grugnog can we get your approval please?

**Comment by dmundra** [comment]
@ContentBrewmaster with #1221 is this one redundant? Also it has some conflicts. Let me know if it is not and I can help resolve the conflicts.

**Pull request closed without merging**